spec/models/story_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe Story do
+  context "creating a new story" do
+
+    let(:attrs) do
+      { hn_story_id:SecureRandom.random_number(8), 
+        title: "Test Title" , 
+        time: '2023-05-03T04:05:06+00:00',
+        by: "Test Author", 
+        url: "wwww.google.come"  }
+    end
+
+    it "should create record" do
+      expect { Story.create(attrs) }.to change{ Story.count }.by(1)
+    end
+  end
+end
```

Comment:
Normally I would create specs for every model, controller, classes, however due to time constraint, I just did this one example to show that I would be using rspec for unit test created. Probably would also utilize FactoryBot for  creation of objects (User, Story, Star) needed for testing